python_modules/dagster/dagster/_core/execution/job_backfill.py

```diff
@@ -90,6 +97,22 @@ def execute_job_backfill_iteration(
             yield None
             time.sleep(CHECKPOINT_INTERVAL)
         else:
+            in_progress_runs = instance.get_runs(
+                RunsFilter(
+                    tags=DagsterRun.tags_for_backfill_id(backfill.backfill_id),
+                    statuses=[
+                        *IN_PROGRESS_RUN_STATUSES,
+                        DagsterRunStatus.QUEUED,
+                        DagsterRunStatus.NOT_STARTED,
```

Comment:
this is basically NOT_FINISHED_STATUSES statuses right? you could add a list in dagster_run.py here much like NON_IN_PROGRESS_RUN_STATUSES: https://github.com/dagster-io/dagster/blob/b88560a0956c8f2d48e3242a02f33135569637ec/python_modules/dagster/dagster/_core/storage/dagster_run.py#L86-L101

Reply:
yeah that makes sense. I'm not totally sure what to do about the MANAGED status though